.gitignore

```diff
@@ -25,10 +25,13 @@ pip-selfcheck.json
 /shapely
 /share
 .ipynb_checkpoints
+notebooks_tmp/
 geodjango.db
 chunked_uploads
 authorized_keys
 
+AlertaDengue/static/geo_info.json
```

Comment:
Is there anything inside static you need track? If not, you can igbore everthing inside static folder